contrib/scalar/scalar.c

```diff
@@ -19,9 +19,27 @@ static int is_unattended(void) {
 	return git_env_bool("Scalar_UNATTENDED", 0);
 }
 
+static void strbuf_parentdir(struct strbuf *buf)
+{
+	int len = buf->len;
+	while (len > 0 && !is_dir_sep(buf->buf[--len]))
+		; /* keep looking for parent directory */
+	strbuf_setlen(buf, len);
+}
+
+/* Given an absolute working directory, find the enlistment root */
+static void strbuf_enlistmentroot(struct strbuf *buf)
+{
+	int len = buf->len;
+	if (len >= 4 && is_dir_sep(buf->buf[len-4]) &&
+			!strncmp("src", buf->buf+len-3, 3))
```

Comment:
nit: add spaces around operators.

```suggestion
	if (len >= 4 && is_dir_sep(buf->buf[len - 4]) &&
			!strncmp("src", buf->buf + len - 3, 3))
```

Reply:
✅